### PR TITLE
doc(zh): sync getting started page at 3ab1e5e

### PR DESCRIPTION
**Pull request by GYHHAHA**
The current Chinese version page for `getting started` is totally different compared to the EN version. User who follow the old introduction steps might get lost or encounter failures, especially using `npm install vue-router@4` to manually add dependency rather than setting everything with `npm create vue@latest`. Hope this will help!

### CODE REVIEW

File: packages/docs/zh/guide/index.md [hunk 1]
```diff
@@ -1,100 +1,206 @@
...
-const app = Vue.createApp({})
-// 确保 _use_ 路由实例使
-// 整个应用支持路由。
-app.use(router)
+`routes` 中定义了一组路由，每个路由把 URL 路径映射到一个组件。其中，由 `component` 参数指定的组件就是先前在 `App.vue` 中被 `<RouterView>` 渲染的组件。这些路由组件通常被称为 _视图_ （views），但本质上它们只是普通的 Vue 组件。
 
-app.mount('#app')
+`routes` 中可以设置的其他选项会在之后介绍，目前我们只需要 `path` 和 `component`。
+
+`history` 选项控制了 `routes` 和 `URL` 路径是如何对应的。在演练场的例子里，我们使用了 `createMemoryHistory()`，它会完全忽略浏览器的 `URL` 而使用其自己内部的 `URL`。 虽然这个例子中是正常工作的，但是这不一定是你想要在实际应用中使用的。通常，你应该使用 `createWebHistory()` 或 `createWebHashHistory()`。我们将在[不同的历史记录模式](./essentials/history-mode)的部分详细介绍这个主题。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:78]:
```suggestion
这里的 `history` 选项控制了路由和 URL 路径是如何双向映射的。在演练场的示例里，我们使用了 `createMemoryHistory()`，它会完全忽略浏览器的 URL 而使用其自己内部的 URL。 这在演练场中可以正常工作，但是未必是你想要在实际应用中使用的。通常，你应该使用 `createWebHistory()` 或 `createWebHashHistory()`。我们将在[不同的历史记录模式](./essentials/history-mode)的部分详细介绍这个主题。
```

File: packages/docs/zh/guide/index.md [hunk 2]
```diff
@@ -1,100 +1,206 @@
...
 ```
 
-通过调用 `app.use(router)`，我们会触发第一次导航且可以在任意组件中以 `this.$router` 的形式访问它，并且以 `this.$route` 的形式访问当前路由：
+和大多数的 Vue 插件一样，`use()` 需要在 `mount()` 之前调用。
+
+如果你好奇这个插件做了什么，它的职责包括：
+
+1. [全局注册](https://vuejs.org/guide/components/registration.html#global-registration) `RouterView` 和 `RouterLink` 组件。
+2. 添加全局 `$router` 和 `$route` 属性。
+3. 启用 `useRouter()` 和 `useRoute()` 函数。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:104]:
```suggestion
3. 启用 `useRouter()` 和 `useRoute()` 组合式函数。
```

File: packages/docs/zh/guide/index.md [hunk 3]
```diff
@@ -1,100 +1,206 @@
...
-      }
+    goToAbout() {
+      this.$router.push('/about')
     },
   },
 }
 ```
 
-要在 `setup` 函数中访问路由，请调用 `useRouter` 或 `useRoute` 函数。我们将在 [Composition API](./advanced/composition-api.md#在-setup-中访问路由和当前路由) 中了解更多信息。
+这里调用了 `push()`，这是用于 [编程式导航](./essentials/navigation)的方法。我们会在后面详细了解。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:127]:
```suggestion
这里调用了 `push()`，这是用于[编程式导航](./essentials/navigation)的方法。我们会在后面详细了解。
```

File: packages/docs/zh/guide/index.md [hunk 4]
```diff
@@ -1,100 +1,206 @@
...
+
+
+对于组合式 API，我们不能通过 `this` 访问组件实例，所以 Vue Router 给我们提供了一些可组合的函数（composables）。演练场示例中的 `AboutView.vue` 组件使用这种方法：
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:130]:
```suggestion
对于组合式 API，我们不能通过 `this` 访问组件实例，所以 Vue Router 给我们提供了一些组合式函数。演练场示例中的 `AboutView.vue` 组件使用了这种方法：
```

File: packages/docs/zh/guide/index.md [hunk 5]
```diff
@@ -1,100 +1,206 @@
...
+  }
+})
+</script>
+```
+
+你现在不一定要完全理解这段代码，关键是要知道可以通过 `useRoute()` 和 `useRouter()` 来访问路由器实例和当前路由。
+
+### 下一步
+
+如果你想要在 Vite 中使用完整的示例，你可以使用 [create-vue](https://github.com/vuejs/create-vue) 工具，它可以在你创建项目的时候将包含 Vue Router 作为选项之一：
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:155]:
```suggestion
如果你想要在 Vite 中使用完整的示例，你可以使用 [create-vue](https://github.com/vuejs/create-vue) 工具，它提供了在项目加入 Vue Router 的选项：
```

File: packages/docs/zh/guide/index.md [hunk 6]
```diff
@@ -1,100 +1,206 @@
...
+
+:::
+
+上述通过 create-vue 创建的示例项目使用了与我们在这里看到的类似的功能，对于探索后续介绍的其他功能而言，也许你会觉得这是一个不错的起点。
+
+## 本教程的约定
+
+### 单文件组件
+
+Vue Router 经常在使用 [Vite](https://vitejs.dev/guide/) 或者 [SFCs](https://vuejs.org/guide/introduction.html#single-file-components) 构建的应用中使用。本教程大多数的例子都在其基础上写成，但是 Vue Router 本身并不需要你使用构建工具或 SFCs。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:179]:
```suggestion
Vue Router 经常在配合打包工具 (如 Vite) 和[单文件组件](https://cn.vuejs.org/guide/introduction.html#single-file-components) (即 `.vue` 文件) 的应用中使用。本教程大多数的示例都是如此，但是 Vue Router 本身并不要求你使用构建工具或单文件组件。
```

File: packages/docs/zh/guide/index.md [hunk 7]
```diff
@@ -1,100 +1,206 @@
...
+
+例如，若你要使用 [Vue](https://vuejs.org/guide/quick-start.html#using-vue-from-cdn) 的全局构建（global builds）和 [Vue Router](../installation#Direct-Download-CDN) ，库将被暴露为全局对象，而不是导入
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:181]:
```suggestion
例如，若你要使用 [Vue](https://cn.vuejs.org/guide/quick-start.html#using-vue-from-cdn) 和 [Vue Router](../installation#Direct-Download-CDN) 的使用全局构建版本，这些库将被暴露为全局对象，而不是导入：
```

File: packages/docs/zh/guide/index.md [hunk 8]
```diff
@@ -1,100 +1,206 @@
...
+
+```js
+const { createApp } = Vue
+const { createRouter, createWebHistory } = VueRouter
+```
+
+### 组件 API 风格
+
+Vue Router 可以使用 Composition API 或 Options API 。在相关部分中，例子将会同时使用两种风格，Composition API 例子通常会使用 `<script setup>`，而不是显式的 `setup` 函数。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:190]:
```suggestion
Vue Router 可以使用组合式 API 或选项式 API 。在必要时，示例将会同时使用两种风格，组合式 API 示例通常会使用 `<script setup>`，而不是显式的 `setup` 函数。
```

File: packages/docs/zh/guide/index.md [hunk 9]
```diff
@@ -1,100 +1,206 @@
...
+
+如果你对于这两种风格不熟悉，可以参考 [Vue - API Styles](https://vuejs.org/guide/introduction.html#api-styles)。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:192]:
```suggestion
如果你对于这两种风格不熟悉，可以参考 [Vue - API 风格](https://cn.vuejs.org/guide/introduction.html#api-styles)。
```

File: packages/docs/zh/guide/index.md [hunk 10]
```diff
@@ -1,100 +1,206 @@
...
+
+### `router` 和 `route`
+
+在本教程中，当提到路由器（router）和路由实例（router instance），它们说的是一种东西，即是 `createRouter()` 返回的对象。在你的应用中，如何访问它们取决于上下文。例如，在 Composition API 中，它可以通过调用 `useRouter()` 来访问。在选项式 API 中，它可以通过 `this.$router` 来访问。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:196]:
```suggestion
在本教程中，我们常常以 `router` 作为路由器实例提及。即由 `createRouter()` 返回的对象。在应用中，访问该对象的方式取决于上下文。例如，在组合式 API 中，它可以通过调用 `useRouter()` 来访问。在选项式 API 中，它可以通过 `this.$router` 来访问。
```

File: packages/docs/zh/guide/index.md [hunk 11]
```diff
@@ -1,100 +1,206 @@
...
+
+类似的，当提到当前路由（current route）和路由（route），它们说的也是一个东西。取决于你的组件使用何种 API 风格，它们可以通过 `useRoute()` 和 `this.$route` 来访问。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:198]:
```suggestion
类似地，当前路由会以 `route` 被提及。基于不同 API 风格的组件，它可以通过 `useRoute()` 或 `this.$route` 来访问。
```

File: packages/docs/zh/guide/index.md [hunk 12]
```diff
@@ -1,100 +1,206 @@
...
+
+### `RouterView` 和 `RouterLink`
+
+组件 `RouterView` 和 `RouterLink` 都是 [全局注册](https://vuejs.org/guide/components/registration.html#global-registration) 的，因此它们不需要在组件模板中导入。但你也可以通过局部导入它们，例如 `import { RouterLink } from 'vue-router'`
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:202]:
```suggestion
组件 `RouterView` 和 `RouterLink` 都是[全局注册](https://cn.vuejs.org/guide/components/registration.html#global-registration)的，因此它们不需要在组件模板中导入。但你也可以通过局部导入它们，例如 `import { RouterLink } from 'vue-router'`。
```

File: packages/docs/zh/guide/index.md [hunk 13]
```diff
@@ -1,100 +1,206 @@
...
+
+在模板中，组件的名字可以是 PascalCase 风格或 kebab-case 风格的。Vue 的模板编译器支持两种格式，因此 `<RouterView>` 和 `<router-view>` 通常是等效的。此时应该遵循你自己项目中使用的约定。
 
-在整个文档中，我们会经常使用 `router` 实例，请记住，`this.$router` 与直接使用通过 `createRouter` 创建的 `router` 实例完全相同。我们使用 `this.$router` 的原因是，我们不想在每个需要操作路由的组件中都导入路由。
+如果你使用 in-DOM 模板，那么需要 [注意](https://vuejs.org/guide/essentials/component-basics.html#in-dom-template-parsing-caveats) ：组件名字必须使用 kebab-case 风格且不支持自闭合标签。因此你不能直接写 `<RouterView />`，而需要使用 `<router-view></router-view>`
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:206]:
```suggestion
如果使用 DOM 内模板，那么需要[注意](https://cn.vuejs.org/guide/essentials/component-basics.html#in-dom-template-parsing-caveats)：组件名字必须使用 kebab-case 风格且不支持自闭合标签。因此你不能直接写 `<RouterView />`，而需要使用 `<router-view></router-view>`。
```

File: packages/docs/zh/guide/index.md [hunk 14]
```diff
@@ -1,100 +1,206 @@
...
+app.use(router)
+app.mount('#app')
 ```
 
-通过调用 `app.use(router)`，我们会触发第一次导航且可以在任意组件中以 `this.$router` 的形式访问它，并且以 `this.$route` 的形式访问当前路由：
+和大多数的 Vue 插件一样，`use()` 需要在 `mount()` 之前调用。
+
+如果你好奇这个插件做了什么，它的职责包括：
+
+1. [全局注册](https://vuejs.org/guide/components/registration.html#global-registration) `RouterView` 和 `RouterLink` 组件。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/index.md:102]:
```suggestion
1. [全局注册](https://cn.vuejs.org/guide/components/registration.html#global-registration) `RouterView` 和 `RouterLink` 组件。
```

**Comment by GYHHAHA** [comment]
@Jinjiang Thanks for your revision! I actually found that most of other pages in EN has already been converted to use `SFC`, I may launch another 2 PRs, respectively for the `Essentials` part and the `Advanced`. Otherwise the contexts for those in ZH could still be misleading.

**Comment by Jinjiang** [comment]
> @Jinjiang Thanks for your revision! I actually found that most of other pages in EN has already been converted to use `SFC`, I may launch another 2 PRs, respectively for the `Essentials` part and the `Advanced`. Otherwise the contexts for those in ZH could still be misleading.

@GYHHAHA good topic. I usually use the full Chinese translation 单文件组件 since I consider this as the most understandable term. And by this chance, I also saw some other places keep using SFC. So I'd like to get this topic into @vuejs-translations/zh-cn team for more discussion. Thanks.

**Pull request merged**